src/mscorlib/src/System/Buffer.cs

```diff
@@ -262,6 +262,15 @@ internal unsafe static void Memmove(byte* dest, byte* src, nuint len)
         {
 #if AMD64 || (BIT32 && !ARM)
             const nuint CopyThreshold = 2048;
+#elif ARM64
+#if PLATFORM_WINDOWS
+            // TODO-ARM64-WINDOWS-OPT determine optimal value for Windows
+            const nuint CopyThreshold = 2048;
+#else // PLATFORM_WINDOWS
+            // Managed code is currently faster than glibc unoptimized memmove
+            // TODO-ARM64-UNIX-OPT revisit when glibc optimized memmove is in Linux distros
+            const nuint CopyThreshold = UInt64.MaxValue;
```

Comment:
Does the JIT dead-code eliminate the PInvoke to memmove?

Reply:
@jkotas I haven't looked yet, but I was hoping it would.  The branch would certainly be predictable.  I'll take a look.

Reply:
@jkotas The compare & branch to the PInvoke IG is not eliminated.  I ran perf tests, there is no measurable difference.  This is believable since the branch is so predictable.

I will leave the dead code unless you have an objection.

Reply:
ok.